### ISSUE TITLE
Remove hmac-ripemd160 from MACs

### ISSUE THREAD

**Issue by LeoVerto** [stribika/stribika.github.io#46]
OpenSSH is planning to remove RIPEMD-160 support along with Blowfish and RC4. [1]

Secure Secure Shell currently recommends including `hmac-ripemd160-etm` and `hmac-ripemd160` in the list of MACs, both of which aren't part of the default list. [2]

Shouldn't Secure Secure Shell no longer recommend these algorithms if they aren't even part of the default configuration anymore?

[1]: https://www.openssh.com/txt/release-7.5
[2]: http://man.openbsd.org/sshd_config.5#MACs

**Comment by madbence** [comment]
fyi, [`ssh-audit`](https://github.com/arthepsy/ssh-audit) (i recently discovered this simple python script) also recommends removing it.

**Comment by Noctem** [comment]
It's been removed now so I just opened a PR for removing it from the page: #49

**Comment by matteoipri** [comment]
I just updated my systems which run archlinux. I followed this guide as suggested in archwiki and I ended up with non working ssh. I confirm that removing the two MACs above solves the issue. `sshd` was complaining of invalid ssh2 MACs specified.

**Comment by graysky2** [comment]
@matteoipri - Yes, the new update causes problems with the currently suggested MACs:
@stribika - You really need to update your guide to reflect the current upstream status... without physical access to the sever, simply updating to openssh 7.6p1 will cause instantaneous breakage and major pain in the balls.

```
fatal: rexec line 21: Bad SSH2 mac spec 'hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-ripemd160-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,hmac-ripemd160,umac-128@openssh.com'.
```

**Comment by jasonkarns** [comment]
This can be closed now, due to 6cb1ee5619d1e4479554fc285156416a9e05f918
